control-plane/agents/src/bin/core/volume/operations.rs

```diff
@@ -611,6 +619,20 @@ impl ResourceShutdownOperations for OperationGuardArc<VolumeSpec> {
     }
 }
 
+/// Checks if Nexus is present in registered target list. Returns true if yes.
+fn filter_targets(registered_target: Vec<String>, nexus: Nexus) -> Result<bool, SvcError> {
```

Comment:
this name is a big vague, how about `target_registered` or something like that?

Reply:
Done